students/mxwllndrsn/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,28 @@
+#uw python 210
+#lesson 02
+#max anderson
+
+# fizzin and a buzzin
+# for fun you can select which multiples get fizzed/buzzed
+# defaults to 3 and 5 per assignment
+
+def fizz_buzz(a=3, b=5):
+
+    for i in range(100):
```

Comment:
you can have your range be `range(1, 101)` to start at 1 (avoid checking for zero) and it will include number 100